### PR TITLE
Add Document.relatedItem

### LINKED ISSUE #816: Add Document.relatedItem field
How frequently are documents related to a specific item?

https://github.com/openprocurement/ocds_linkedDocument_extension adds Document.relatedItem (string) and Document.documentScope (string). I am curious if other publishers need this, too.

Enquiry and Criterion from the enquiries and requirements extensions, respectively, already have a `relatedItem` field.

cc @yanakapatsila

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes #816

### CODE REVIEW

**Comment by duncandewhurst** [comment]
One question to resolve in https://github.com/open-contracting/standard/issues/816#issuecomment-943838042

**Review by mrshll1001 — APPROVED** [review]

File: docs/history/changelog.md [hunk 1]
```diff
@@ -167,10 +167,11 @@ Per the [normative and non-normative content and changes policy](https://docs.go
     * `contracts.estimatedValue`
   * [#1372](https://github.com/open-contracting/standard/pull/1372) `Address.country`
   * [#1380](https://github.com/open-contracting/standard/pull/1380) `Document.languages`
+  * [#1433](https://github.com/open-contracting/standard/pull/1433) `Document.relatedItems`
 
 * Deprecate some fields:
   * [#1200](https://github.com/open-contracting/standard/pull/1200) `tender.submissionMethod`, because all codes from the `submissionMethod` codelist are deprecated.
-  * [#1296](https://github.com/open-contracting/standard/pull/1296) `tender.eligibilityCriteria` in favor of the new `tender.exclusionGrounds` field, in order to use more common terminology and improve semantics. 
+  * [#1296](https://github.com/open-contracting/standard/pull/1296) `tender.eligibilityCriteria` in favor of the new `tender.exclusionGrounds` field, in order to use more common terminology and improve semantics.
```
Comment (jpmckinney) [inline docs/history/changelog.md:174]: Can you disable whatever option in your editor is changing line endings? Alternatively, change editor. Please re-do the PR from a new, clean branch, but without these whitespace changes.

File: schema/release-schema.json [hunk 2]
```diff
@@ -1129,6 +1129,19 @@
             "description": "This field is deprecated in favor of `languages`, to support documents in which multiple languages are used.",
             "deprecatedVersion": "1.2"
           }
+        },
+        "relatedItems": {
+          "title": "Related items",
+          "description": "If the document relates to one or more items, the IDs of the related item(s) in the items array of the parent object. For example, for a document in `tender.documents`, the `.id` of the related item(s) in `tender.items`.",
```
Comment (jpmckinney) [inline schema/release-schema.json]: The "if" clause is not needed, as this is true for any field. The "(s)" construction would have to be "the ID(s) of the related item(s)", but this is unnecessarily awkward to read. The plural is fine (like the description of `documents` itself). The semantics are more properly the related items, not their identifiers – their identifiers are instead just a technical (not semantic) detail.

An important distinction to make is that this field is only relevant if the document applies to a subset of items.

```suggestion
          "description": "The subset of items to which this document relates, expressed as the `id` values of the related entries in the parent object's `items` array. For example, an entry in the `tender.documents` array may be related only to entries in the `tender.items` array.",
```

File: docs/history/changelog.md (hunk 1 above)
Comment (duncandewhurst) [reply docs/history/changelog.md:174]: Well I have no idea why Atom is doing this :-(

image

Comment (duncandewhurst) [reply docs/history/changelog.md:174]: https://github.com/open-contracting/standard/pull/1439/

**Pull request closed without merging**